client/analytics/report/downloads/table.js

```diff
@@ -119,14 +122,20 @@ export default class CouponsReportTable extends Component {
 		if ( ! totals ) {
 			return [];
 		}
+		const { query } = this.props;
+		const dates = getCurrentDates( query );
+		const after = moment( dates.primary.after );
+		const before = moment( dates.primary.before );
+		const days = before.diff( after, 'days' );
```

Comment:
Should this be `const days = before.diff( after, 'days' ) + 1;` to include the start date?

For example, if we select Jan 2 - Jan 2, this shows 0 days.

Reply:
Of course! Good catch!